README.md

```diff
@@ -1 +1,82 @@
-# covid-data
\ No newline at end of file
+# React Capstone Project: Covid Data Website
+
+This is a capstone project for Module 3-React. This website is mobile-friendly and has two pages: Home and Details. The home page lists the 194 countries with the COVID summary. The Details page shows the detailed COVID data of each country selected.
+
+# Screen Shots
+
+## Mobile
+
+<p align="center">
+<img src="https://user-images.githubusercontent.com/49184579/169946799-e5123043-4387-4ab2-8e23-e81240f1e945.png" width="360" >
+</p>
+
+<p align="center">
+<img src="https://user-images.githubusercontent.com/49184579/169946813-1c0b4f77-6755-4987-a211-f1575cbd139c.png" width="360" >
+</p>
+
+## Built With &hearts;
+
+- ReactJS
+- Redux
+- Redux-thunk
+- Axios
+- Bootstrap
+
+## Live Demo
+
+[Github-page live](https://aboongm.github.io/covid-data/)
+
+## Video Link On Youtube
+
+[Youtube](https://youtu.be/qS4A8aZegmo)
+
+### Prerequisites
+
+- Good knowledge on javascript objects
+- VSCode or other text editor installed. [Link to download VSCode](https://code.visualstudio.com/download)
+- Node package. [Link to download node](https://nodejs.org/en/download/)
+
+## Getting started
+
+to run the project locally, clone the project using the command
+
+`git clone git@github.com:aboongm/covid-data.git`
+
+### Run test
+
+navigate to the project directory, from GitHub,
+to test or run the application double click on the `index.html` file to open in your browser
+
+## Authors
+
+👤 **Ranjit Luwang**
+
+- GitHub: [@aboongm](https://github.com/aboongm)
+- Twitter: [@John_luang1](https://twitter.com/John_luang1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/aboongm)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Microverse program
+- My standup team
+- Microverse team
+- My family's support
+
```

Comment:
- [ ] Kindly add acknowledgment of the original design creator and a link to that design to give credit to the designer. 👍